### PR TITLE
Create release v0.42.0

### PR DESCRIPTION
**Pull request by metalmatze**
> The Answer to the Ultimate Question of Life, the Universe, and Everything is 42

Let me know if the changelog sounds about right.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 ## Next release
 
+## 0.42.0 / 2020-09-09
+
+Most importantly the Prometheus Operator now lives in its own indepent GitHub organization.
+
+* [CHANGE] Introduce initial governance (#3398)
```
Comment (simonpasquier) [inline CHANGELOG.md:7]: I would move it to the introduction with the new org.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+* [ENHANCEMENT] Instrument client-go requests (#3465)
+* [ENHANCEMENT] pkg/prometheus: skip invalid service monitors (#3445)
+* [ENHANCEMENT] pkg/alertmanager: Use lower value for --cluster.reconnect-timeout (#3436)
+* [ENHANCEMENT] pkg/alertmanager: cleanup resources via OwnerReferences (#3423)
+* [ENHANCEMENT] Add prometheus_operator_reconcile_operations_total metric (#3415)
+* [ENHANCEMENT] pkg/operator/image.go: Adjust image path building (#3392)
+* [ENHANCEMENT] Specify timeouts per Alertmanager target when sending alerts. (#3385)
+* [ENHANCEMENT] Push container images to Docker Hub and Quay (#3390)
+* [ENHANCEMENT] Run single replica Alertmanager in HA cluster mode (#3382)
+* [ENHANCEMENT] decontainerize shellcheck (#3379)
```
Comment (simonpasquier) [inline CHANGELOG.md:19]: AFAIU it's not user visible so I wouldn't mention it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+* [BUGFIX] Fix validation logic for SecretOrConfigMap (#3413)
+* [BUGFIX] Don't overwrite __param_target (#3377)
```
Comment (s-urbaniak) [inline CHANGELOG.md:21]: I don't see https://github.com/prometheus-operator/prometheus-operator/pull/3440 in here. Doesn't it make sense to add it as it fixes potential reconciliation issues?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+* [FEATURE] Move API types out into their own module (#3395)
+* [FEATURE] Create a monitoring mixin for prometheus-operator (#3333)
+* [ENHANCEMENT] Instrument client-go requests (#3465)
+* [ENHANCEMENT] pkg/prometheus: skip invalid service monitors (#3445)
+* [ENHANCEMENT] pkg/alertmanager: Use lower value for --cluster.reconnect-timeout (#3436)
+* [ENHANCEMENT] pkg/alertmanager: cleanup resources via OwnerReferences (#3423)
+* [ENHANCEMENT] Add prometheus_operator_reconcile_operations_total metric (#3415)
+* [ENHANCEMENT] pkg/operator/image.go: Adjust image path building (#3392)
+* [ENHANCEMENT] Specify timeouts per Alertmanager target when sending alerts. (#3385)
+* [ENHANCEMENT] Push container images to Docker Hub and Quay (#3390)
```
Comment (simonpasquier) [inline CHANGELOG.md:17]: Hmm I don't think the image is on DockerHub.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,25 @@
 ## Next release
 
+## 0.42.0 / 2020-09-09
+
+Most importantly the Prometheus Operator now lives in its own indepent GitHub organization.
```
Comment (simonpasquier) [inline CHANGELOG.md:5]: It had already happened for 0.41?

File: CHANGELOG.md (hunk 3 above)
Comment (metalmatze) [reply CHANGELOG.md:21]: Totally, seems like that last PR, and I, for some reason, didn't add. I guess I was too eager to get it out...
Adding.

File: CHANGELOG.md (hunk 5 above)
Comment (metalmatze) [reply CHANGELOG.md:5]: I don't think so. All PRs about moving were between [v0.41.1...master](https://github.com/prometheus-operator/prometheus-operator/compare/v0.41.1...master).

File: CHANGELOG.md (hunk 4 above)
Comment (metalmatze) [reply CHANGELOG.md:17]: Right. I didn't get that it seems. Here's what I rephrased to:
> Push container images to Quay into coreos and prometheus-operator orgs

**Comment by metalmatze** [comment]
New PR lives here:
https://github.com/prometheus-operator/prometheus-operator/pull/3483

**Pull request closed without merging**